src/main/pages/che-7/installation-guide/proc_installing-che-using-storage-classes.adoc

```diff
@@ -0,0 +1,204 @@
+// Module included in the following assemblies:
+//
+// advanced-configuration-options
+
+[id="proc_installing-{prod-id-short}-using-storage-classes_{context}"]
+ 
+= Installing {prod-short} using storage classes
+ 
+:context: proc_installing-{prod-id-short}-using-storage-classes
```

Comment:
The context should not include "proc_".
```suggestion
:context: installing-{prod-id-short}-using-storage-classes
```

Reply:
Just realized... this a module included from an assembly. It shouldn't set its own context at all.
```suggestion
```